setup.sh

```diff
@@ -105,6 +105,12 @@ echo "Setting up TnP tools..."
 #git cms-addpkg DataFormats/RecoCandidate
 #git cms-addpkg PhysiscsTools/TagAndProbe
 git cms-merge-topic -u matteosan1:egm_tnp_76X_v1
+git cms-merge-topic -u matteosan1:smearer_76X_test
+git remote add cmssw-ferriff https://github.com/ferriff/cmssw.git
+git fetch cmssw-ferriff
```

Comment:
Yes, I'll try to do this later this week.  But I have tested this setup.sh file at IC and it works ok.